admin_console/controllers/PartnerController.php

```diff
@@ -579,6 +579,22 @@ private function getPartnerFilterFromRequest(Zend_Controller_Request_Abstract $r
 			$filterType = 'byid';
 			
 		}
+		if($filterType == 'byUIConfId') {
+		    $client = Infra_ClientHelper::getClient();
+		    $adminConsolePlugin = Kaltura_Client_AdminConsole_Plugin::get($client);
+		
+		    try {
+		        $uiConf = $adminConsolePlugin->uiConfAdmin->get($filterInput);
+		        /* @var $uiConf Kaltura_Client_Type_UIConf  */
+		        $filterInput = $uiConf->partnerId;
+		    }
+		    catch(Exception $ex) {
+		        $filterInput = "-99";
+		    }
+		    $filterType = 'byid';
```

Comment:
You should keep the filter type and input for the UI, just add the partner id to $filter->idIn instead.